<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border-radius:20px;
background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QToolButton" name="serverToolButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
  color: rgb(0, 0, 0);
}

QToolButton {
  background-color: transparent;
  border: none;
}

QToolButton:checked {
  color: rgb(0, 85, 255);
  border: 2px solid rgb(0, 85, 255);
  border-radius: 4px; /* 可选：添加圆角 */
}

/* 可选：添加悬停效果 */
QToolButton#serverToolButton:hover {
  background-color: rgba(0, 85, 255, 0.1);
}</string>
   </property>
   <property name="text">
    <string>服务设置</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="instrumentToolButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
  color: rgb(0, 0, 0);
}

QToolButton {
  background-color: transparent;
  border: none;
}

QToolButton:checked {
  color: rgb(0, 85, 255);
  border: 2px solid rgb(0, 85, 255);
  border-radius: 4px; /* 可选：添加圆角 */
}

/* 可选：添加悬停效果 */
QToolButton#instrumentToolButton:hover {
  background-color: rgba(0, 85, 255, 0.1);
}</string>
   </property>
   <property name="text">
    <string>仪器设置</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>511</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="ipEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid gray;
padding-left:5px;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>请输入服务器IP地址</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>域名</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>38</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid gray;
padding-left:5px;
font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>请输入服务器域名，如：http://bk4.ilabeco.com</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>481</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* { border: none; }
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyWord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>391</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(194, 194, 194);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入关键字</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchMachine">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font: 12pt &quot;微软雅黑&quot;;
	
	color: rgb(136, 136, 136);
	border-radius: 5px;


	background-color: rgb(255, 255, 255);
	border: 1px solid gray;

}
QPushButton:hover{
	
	color: rgb(0, 85, 255);
	border: 1px solid blue;

}

QPushButton::checked {
	

	border: 1px solid blue;
	color: rgb(255, 255, 255);

}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>填写说明：请填写需要随程序关闭的进程名，以逗号隔开！</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="taskTextEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>451</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="tackCheck">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>330</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>是否启用</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>401</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{	
	color: rgb(136, 136, 136);
	border-radius: 5px;
	margin-left:5px;
	padding:10px 20px;
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;


}
QPushButton:hover{
	color: rgb(0, 85, 255);
	border: 1px solid blue;
}

QPushButton::checked {
	border: 1px solid blue;
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>401</y>
     <width>185</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(136, 136, 136);
	border-radius: 5px;
	margin-left:5px;
	padding:10px 20px;
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;

}
QPushButton:hover{
	
	color: rgb(0, 85, 255);
	border: 1px solid blue;

}

QPushButton::checked {
	
	background-color: blue;
	border: 1px solid blue;
	color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QToolButton" name="otherToolButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
  color: rgb(0, 0, 0);
}

QToolButton {
  background-color: transparent;
  border: none;
}

QToolButton:checked {
  color: rgb(0, 85, 255);
  border: 2px solid rgb(0, 85, 255);
  border-radius: 4px; /* 可选：添加圆角 */
}

/* 可选：添加悬停效果 */
QToolButton#otherToolButton:hover {
  background-color: rgba(0, 85, 255, 0.1);
}</string>
   </property>
   <property name="text">
    <string>其它设置</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
